src/controller.ts

```diff
@@ -3,22 +3,25 @@ import { dirname, join } from 'path';
 import { clearDecorations, setLinesHeat } from './view';
 import { absolutePath, AustinStats } from './model';
 import { getAustinCommand } from './utils/commandFactory';
+import { isPythonExtensionAvailable } from './utils/pythonExtension';
 
 
 function delay(ms: number) {
     return new Promise(resolve => setTimeout(resolve, ms));
 }
 
 export class AustinController {
+    private output: vscode.OutputChannel = vscode.window.createOutputChannel("Austin");
 
-    public constructor(private stats: AustinStats, private output: vscode.OutputChannel) { }
+    public constructor(private stats: AustinStats, ) { }
 
     public async profileScript() {
         const currentUri = vscode.window.activeTextEditor?.document.uri;
-
+        if (!isPythonExtensionAvailable()) {
+            throw Error("Python extension not available");
```

Comment:
Does this show a notification popup in VS Code? Should the message then be more actionable by the user (e.g. specify the exact extension, or even better have a clickable link if possible)?

Reply:
I can't actually get this to fail anyway, it waits for the extension to be loaded before allowing the command to be run, you can't disable the Python extension manually, because its a dependency of Austin. This error code would be shown to the user, but I don't know under what circumstance.